### DISCUSSION TITLE
Help with Autologin

### DISCUSSION THREAD

**Discussion by Detla7t** [ValvePython/steam#393] [Q&A] [answered]
I don't know if you are suppose to be able to but quick question i was looking at this [recipe](https://github.com/ValvePython/steam/blob/8ce3b5041450ef9ae0b28f7138d865d9a3d82e48/recipes/1.Login/persistent_login.py) and was curious if you can use the sentry file that is made to login automatically kinda like a cookie? if not i was just going to setup steam auth but i would prefer not if possible. thanks :)

**Comment by rossengeorgiev** [comment, marked as answer]
https://steam.readthedocs.io/en/stable/api/steam.client.html?highlight=relogin#steam.client.SteamClient.relogin

You need to store the `username` and `login_key`, then use those to login. No email or 2FA will be required. You must set the credentials location for this work. This is essentially Steam's remember password feature.

The recipe needs to be updated to show that part of the process. 

Here is an example:

https://github.com/ValvePython/steamctl/blob/9114df53c4c56ad8c768bdca196f8a1f2bc483ef/steamctl/clients.py#L33-L37
https://github.com/ValvePython/steamctl/blob/9114df53c4c56ad8c768bdca196f8a1f2bc483ef/steamctl/clients.py#L65-L77
